cantabular/dimensions_test.go

```diff
@@ -68,8 +68,12 @@ func TestGetDimensionsUnhappy(t *testing.T) {
 		Convey("When GetDimensions is called", func() {
 			resp, err := cantabularClient.GetDimensions(testCtx, "Teaching-Dataset")
 
+			// Convey("Then the expected error is returned", func() {
```

Comment:
Lots of tests still commented out, you probably want to replace with `So(err, ShouldNotBeNil)` in addition to the statusCode check you've added for these

Reply:
bear with me on this one, ive been working on this as we have been pair debugging the coverage ticket, and just experimenting with the broken tests just to get them working, ill ping you when its cleaned up